src/app/components/challenge/challengeleaderboard/challengeleaderboard.component.ts

```diff
@@ -23,6 +23,16 @@ export class ChallengeleaderboardComponent implements OnInit, AfterViewInit {
   @ViewChildren('phasesplitselect')
   components: QueryList<SelectphaseComponent>;
 
+  /**
+   * Leaderboard Precision Value
+   */
+  LeaderboardPrecisionValue = 2;
+
+    /**
+   * Challenge phase split pk Value
+   */
+  challenge_phase_Split_pk;
```

Comment:
Please change the comment to `Challenge phase split ID` and variable name to `challengePhaseSplitId`